client/web/src/nav/StatusMessagesNavItem.tsx

```diff
@@ -424,26 +434,32 @@ export class StatusMessagesNavItem extends React.PureComponent<Props, State> {
         ) {
             return (
                 <Icon
+                    role="img"
                     data-tooltip={this.state.isOpen ? undefined : 'Syncing repositories failed!'}
                     as={CloudAlertIconRefresh}
                     size="md"
+                    aria-hidden={true}
```

Comment:
```suggestion
                    title={tooltip}
```

Same as `data-tooltip`